tests/components/zwave_js/test_light.py

```diff
@@ -859,3 +861,143 @@ async def test_black_is_off_zdb5100(
         "property": "targetColor",
     }
     assert args["value"] == {"red": 255, "green": 76, "blue": 255}
+
+
+async def test_basic_light(
```

Comment:
The terminology "basic light" is a bit misleading imo. Better use "basic_cc_light" which is more describing